ThatValidator.js

```diff
@@ -362,10 +347,10 @@
 
                 for(var i = 0; i < elements.length; i++)
                 {
-                    var field = elements[i];
+                    var field2 = elements[i];
 
                     //if the element has not already been found
-                    if(self.fields.indexOf(field) == -1)
+                    if(self.fields.indexOf(field2) == -1)
```

Comment:
I fixed the code. Do i need to resubmit the pull request, or will it 
magically update itself?


On 9/14/2015 4:26 PM, Jonathan Dumaine wrote:
>
> In ThatValidator.js 
> <https://github.com/JonDum/ThatValidator/pull/2#discussion_r39442251>:
>
> >
> >                      //if the element has not already been found
> > -                    if(self.fields.indexOf(field) == -1)
> > +                    if(self.fields.indexOf(field2) == -1)
>
> I think you did it in the wrong place then. I see what you're talking 
> about in the validate() function, but this is in the init() function
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/JonDum/ThatValidator/pull/2/files#r39442251>.
>